tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
```

Comment:
For the moment the file contain some operations but the ```date``` library will grow ! (same think to the file test)